### DISCUSSION TITLE
Use turbo without cache

### DISCUSSION THREAD

**Discussion by skofgar** [vercel/turborepo#7243] [Help] [answered]
Hello,

I'm getting different errors on different machines and in CI. I'm trying to debug turbo and am attempting to run it without cache. However, `--no-cache` does not seem to do anything (e.g. directly invoking turbo: `time npx turbo --no-cache build` returns in `0.5s`, although building the project takes much longer... ~`30s`). We have a monorepo, so deleting all the turbo directories is tedious. Is there a command to reset all the turbo caches? 

It seems like that would be a helpful feature to have for testing and debugging, but I couldn't find anything like that in the documentation on https://turbo.build

**Comment by anthonyshew** [comment, marked as answer]
It sounds like you're looking for the `--force` flag: https://turbo.build/repo/docs/reference/command-line-reference/run#--force

**Reply by skofgar** [reply]
ohh 🤦 I overlooked that one, thank you!
